app/src/main/java/ch/epfl/sdp/blindly/profile/Profile2.kt

```diff
@@ -0,0 +1,39 @@
+package ch.epfl.sdp.blindly.profile
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import ch.epfl.sdp.blindly.R
+
+
+private const val min_length = 2
+private const val max_length = 20
```

Comment:
Might be a good idea to rename these constants using capital letters

Reply:
Corrected in the final commit.